addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -627,14 +627,23 @@ export class Rtc extends Record {
         this.store.fullscreenChannel = null;
         this.state.hasPendingRequest = true;
         await this.rpcLeaveCall(channel);
-        this.endCall(channel);
+        this.endCall({ channel });
         this.state.hasPendingRequest = false;
     }
 
     /**
      * @param {import("models").Thread} [channel]
+     * @param {boolean} [isEndedByServer]
      */
-    endCall(channel = this.state.channel) {
+    endCall({ channel = this.state.channel, isEndedByServer = false } = {}) {
```

Comment:
To follow stable policy normally should not make breaking change to function signature.

I'd keep `endCall(channel)` as positional argument, and then support new param with either with 2nd parameter being optional as `options = { isEndedByServer }` or the 1st param is handled as either `{ channel, isEndedByServer }` or `channel`, depending on whether the 1st param is a record or not.